module/VuFind/tests/integration-tests/src/VuFindTest/Mink/ContainerLinksTest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/**
+ * Mink test class for simple container links functionality (including collection routing).
+ *
+ * PHP version 7
+ *
+ * Copyright (C) The National Library of Finland 2023.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/indexing:hierarchies_and_collections Hierarchies and Collections
+ */
+
+namespace VuFindTest\Mink;
+
+use Behat\Mink\Element\Element;
+
+/**
+ * Mink test class for simple container links functionality (including collection routing).
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/indexing:hierarchies_and_collections Hierarchies and Collections
+ */
+class ContainerLinksTest extends \VuFindTest\Integration\MinkTestCase
+{
+    /**
+     * Go to a collection page.
+     *
+     * @return Element
+     */
+    protected function goToCollection()
+    {
+        $session = $this->getMinkSession();
+        $path = '/Collection/topcollection1';
+        $session->visit($this->getVuFindUrl() . $path);
+        return $session->getPage();
+    }
+
+    /**
+     * Test simple container linking without ID.
+     *
+     * @return void
+     */
+    public function testSimpleContainerLinksWithoutID(): void
```

Comment:
You don't have simple container links turned on in this test -- I'm wondering if the test name is misleading. What about testDefaultContainmentBehavior, with a comment like "Test handling of container fields when simpleContainerLinks is false (default behavior)."?

Reply:
I changed it to testDefaultContainerLinks with the suggested comment. Does it work for you?

Reply:
Yep, that's great!